src/util/ExpressionUtils.cpp

```diff
@@ -236,10 +243,89 @@ Status ExpressionUtils::checkAggExpr(const AggregateExpression* aggExpr) {
     return Status::OK();
 }
 
-bool ExpressionUtils::isEvaluableExpr(const Expression* expr) {
-    EvaluableExprVisitor visitor;
-    const_cast<Expression*>(expr)->accept(&visitor);
-    return visitor.ok();
+std::unique_ptr<Expression> ExpressionUtils::reverseRelExpr(Expression *expr) {
+    auto left = static_cast<RelationalExpression *>(expr)->left();
```

Comment:
dont' need DCHECK here?

Reply:
You are right. Thanks for catching this.